client/app/components/sections/Topbar/Topbar.js

```diff
@@ -22,14 +22,27 @@ import MenuMobile from '../../composites/MenuMobile/MenuMobile';
 import MenuPriority from '../../composites/MenuPriority/MenuPriority';
 import SearchBar from '../../composites/SearchBar/SearchBar';
 
-const profileDropdownActions = function profileDropdownActions(routes, username) {
+
+const LABEL_TYPE_MENU = 'menu';
+const LABEL_TYPE_DROPDOWN = 'dropdown';
+const SEARCH_PARAMS_TO_KEEP = ['view', 'locale'];
+const SEARCH_PARAMS = ['q', 'lq'];
+const DEFAULT_CONTEXT = {
+  marketplace_color1: styleVariables['--customColorFallback'],
+  marketplace_color2: styleVariables['--customColor2Fallback'],
```

Comment:
From current version, we might use it later though.
But I'll remove it from topbar.js related code.